<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PLSDialogView</class>
 <widget class="QDialog" name="PLSDialogView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="PLSTransparentForMouseEventsFrame" name="plsDialogViewBorderFrame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="mainLayout" stretch="0,0,1">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="PLSTransparentForMouseEventsFrame" name="titleBar">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="excludeSpecChildren" stdset="0">
         <stringlist notr="true">
          <string>min</string>
          <string>maxres</string>
          <string>close</string>
         </stringlist>
        </property>
        <layout class="QHBoxLayout" name="titleBarLayout" stretch="0,0,1,0">
         <property name="spacing">
          <number>5</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item alignment="Qt::AlignHCenter">
          <widget class="PLSTransparentForMouseEventsLabel" name="titleLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>DialogView</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QPushButton" name="helpBtn">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="dialogRightSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="titleButtonLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QToolButton" name="min">
             <property name="cursor">
              <cursorShape>ArrowCursor</cursorShape>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="maxres">
             <property name="cursor">
              <cursorShape>ArrowCursor</cursorShape>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="close">
             <property name="cursor">
              <cursorShape>ArrowCursor</cursorShape>
             </property>
             <property name="text">
              <string notr="true"/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="hline" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>1</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>1</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="PLSTransparentForMouseEventsFrame" name="content">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="excludeChildren" stdset="0">
         <string notr="true">All</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PLSTransparentForMouseEventsFrame</class>
   <extends>QFrame</extends>
   <header>PLSTransparentForMouseEvents.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PLSTransparentForMouseEventsLabel</class>
   <extends>QLabel</extends>
   <header>PLSTransparentForMouseEvents.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
